application/server/controllers/users.go

```diff
@@ -196,11 +174,32 @@ func (repository *Repos) UpdateUser(c *gin.Context) {
 	if updateUser.Email != "" {
 		user.Email = updateUser.Email
 	}
-	//Note: cannot set IsAdmin if the authorizing user is not an admin
 
 	repository.UserDb.Save(&user)
-	//TODO: uncomment the below line once admin functionality is added
-	//}
-	//TODO: write case for if the authorizing user is an admin in Sprint 4
+
 	c.JSON(http.StatusAccepted, user)
 }
+
```

Comment:
Haven't actually started implementing any admin stuff yet; this is just some stuff that I removed from UpdateUser but wanted to keep for later in Sprint 4